cli/cmd/resource_groups.go

```diff
@@ -0,0 +1,291 @@
+//
+// Author:: Darren Murray(<darren.murray@lacework.net>)
+// Copyright:: Copyright 2021, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/lacework/go-sdk/api"
+	"github.com/olekukonko/tablewriter"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// resource_groups command is used to manage lacework resource groups
+	resourceGroupsCommand = &cobra.Command{
+		Use:     "resource-group",
+		Aliases: []string{"group", "rg"},
```

Comment:
I am not sure that we should alias to `group`, but I could be convinced otherwise
```suggestion
		Aliases: []string{"resource-groups", "rg"},
```

Reply:
Yeh, I'm happy to leave it in favour of rg